src/components/atoms/button/index.js

```diff
@@ -1,4 +1,4 @@
-import React from 'react'
+import React, { Fragment } from 'react'
```

Comment:
We can't use fragments because we want to support React 15 as well 😄 (without polyfills if we can help it)

Reply:
@siddharthkp, I did a little bit of googling and I found that the array syntax you are currently using is supported with React 16 onwards. See https://reactjs.org/blog/2017/09/26/react-v16.0.html#new-render-return-types-fragments-and-strings

The `Fragment` feature is added in React 16.2 .